forum_leaderboard/static/js/src/forum_leaderboard_studio.js

```diff
@@ -0,0 +1,21 @@
+/* Javascript for ForumLeaderboardXBlock. */
+function ForumLeaderboardStudioXBlock(runtime, element) {
+    $('.save-button', element).bind('click', function() {
+        var handlerUrl = runtime.handlerUrl(element, 'studio_submit');
+        var data = {
+            discussion_id: $(element).find('input[name=discussion_id]').val(),
+            count: $(element).find('input[name=count]').val()
+        };
+        $.post(handlerUrl, JSON.stringify(data)).success(function(response) {
+            if (response['success']) {
+                window.location.reload(false);
+                return
+            }
+            alert(response['errors'].join('\n'));
```

Comment:
@Kelketek Is this because XBlocks does not support validation messages workflow? `alert`ing errors are kind of ugly, so there might be a better ways to do that (e.g. discussion displays a styled modal - might be a good idea to reuse or draw inspiration from it).